crates/ef-testing/src/lib.rs

```diff
@@ -39,3 +41,83 @@ pub fn get_signed_rlp_encoded_transaction(
 
     Ok(out.to_vec().into())
 }
+
+/// returns whether a given test at a given path should be ignored or not
+pub fn should_skip(path: &Path) -> bool {
+    let path_str = path.to_str().expect("Path is not valid UTF-8");
+    let name = path.file_name().unwrap().to_str().unwrap();
+
+    // list of files that we do want to include even though their parent directories might be marked for skip
+    let include_files = [
```

Comment:
Since `BlockchainTests`s are skipped, a special array of files can be considered as an exception.

When a filepath matches them, we ignore the skip test as we know they should not be skipped.